apps/cli/src/constants.ts

```diff
@@ -0,0 +1,33 @@
+import { join } from "node:path";
+import envPaths from "env-paths";
+
+const APP_CONFIG = envPaths("webstudio").config;
+const GLOBAL_CONFIG_FILE_NAME = "webstudio-config.json";
+export const GLOBAL_CONFIG_FILE = join(APP_CONFIG, GLOBAL_CONFIG_FILE_NAME);
+
+export const CLI_ARGS_OPTIONS = {
+  debug: {
+    type: <const>"boolean",
```

Comment:
We do not use such syntax caz it is conflicts with jsx. Better do this instead
`export const CLI_ARGS_OPTIONS = { ... } as const`